skimage/color/tests/test_adapt_rgb.py

```diff
@@ -17,76 +18,190 @@
 
 @adapt_rgb(each_channel)
 def edges_each(image):
+    """
+    Applies Sobel edge detection to an RGB image, using each color channel
+    independently.
+
+    Parameters
+    ----------
+    image : ndarray
+        Input RGB image.
+
+    Returns
+    -------
+    result : ndarray
+        Edge map of the input image, with each color channel processed
+        independently.
+    """
     return filters.sobel(image)
 
 
 @adapt_rgb(each_channel)
 def smooth_each(image, sigma):
+    """
+    Applies Gaussian smoothing to an RGB image, using each color channel
+    independently.
+
+    Parameters
+    ----------
+    image : ndarray
+        Input RGB image.
+    sigma : float
+        Standard deviation of the Gaussian kernel.
+
+    Returns
+    -------
+    result : ndarray
+        Smoothed version of the input image, with each color channel processed
+        independently.
+    """
     return filters.gaussian(image, sigma)
 
 
 @adapt_rgb(each_channel)
 def mask_each(image, mask):
+        """
+    Applies a binary mask to an RGB image, with each color channel processed
+    independently.
+
+    Pixels where `mask` is True will be set to zero.
+
+    Parameters
+    ----------
+    image : ndarray
+        Input RGB image.
+    mask : ndarray, bool
+        Boolean mask to apply to the image.
+
+    Returns
+    -------
+    result : ndarray
+        Copy of the input image with masked pixels set to zero.
+    """
```

Comment:
This docstring already looks quite close to the [numpydoc style guide](https://numpydoc.readthedocs.io/en/latest/format.html) that we follow in scikit-image. There are a few things I would change:

- Remove the indent before `"""` in the first line
- Shorten the summary at the beginning to a single line and use the imperative mood (as if you where telling code what to do). For example, "Mask each color channel of an RGB image independently". 
- Change `mask : ndarray, bool` to `mask : ndarray`.

You could also try updating the other docstrings accordingly.

Reply:
sure , i will keep these points in mind and update accordingly

Reply:
Thank you. :) I see that you added some of the changes but I think you missed a few things. For example, in [`edges_hsv_uint`](https://github.com/scikit-image/scikit-image/pull/6776/files#diff-bddd0373044a3a2fd26d4b1ec3603ce7a220c4f7806de5ac307caa19efa021e9R123-R125) and you could try to shorten the summaries to one line (see my third point above). 

Apart from that it looks perfect!